### DISCUSSION TITLE
Ribbon Cable Parts

### DISCUSSION THREAD

**Discussion by neaden** [TillFleisch/ESPHome-Philips-Smart-Coffee#61] [Q&A] [answered]
I found discussion #51 - but I'm having a tough time finding a ribbon cable male to female (so I don't have to cut the existing ribbon cable)

Do you have any recommendations?

**Comment by TillFleisch** [comment, marked as answer]
I have also found that these are difficult to come by.
The part number you're looking for is: 9231508xx, where xx is the length.
The product info is available [here](
https://www.molex.com/en-us/products/part-detail/923150808?display=pdf) and technical drawings which include more information regarding the cable length/product number are available [here](https://www.molex.com/content/dam/molex/molex-dot-com/products/automated/en-us/salesdrawingpdf/923/92315/923150808_sd.pdf). Some sellers separate the last 4 digits.

You can also try to find individual connectors (part number: [903270308](https://www.molex.com/en-us/products/part-detail/903270308?display=pdf))  and attempt to build your own cables using some spare ribbon cable. I have tried this myself, since my first PCB prototype had a reversed connector. I ended up building an 'illegal cable' which reversed the pins. Since I didn't have crimping pliers I used a small screwdriver to push the pins into the cable, which worked well but wasn't a pleasant experience as I managed mangle some pins.


As far as I am aware there are only male connectors (i.e. 903250008) which are mounted to some PCB and cables which clamp onto them (both ends female).
